app/mailers/report_mailer.rb

```diff
@@ -1,30 +1,32 @@
 class ReportMailer < ActionMailer::Base
-  default :from => AppConfig.mail.sender_address
+  default from: AppConfig.mail.sender_address
 
-  def new_report(type, id)
+  def self.new_report(type, id)
+    [Role.admins + Role.moderators].flatten.map {|role| super(type, id, role) }
```

Comment:
You can use Enumerable#flat_map here which is more concise. Bonus points: it's significantly faster as well -> https://github.com/JuanitoFatas/fast-ruby#enumerablemaparrayflatten-vs-enumerableflat_map-code